### ISSUE TITLE
Add support for dynamic locale

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/beers-law-lab#288]
Requested by @kathy-phet.

This also includes Concentration.

**Comment by pixelzoom** [comment]
Baselines published before starting this work:

https://phet-dev.colorado.edu/html/beers-law-lab/1.7.0-dev.5
https://phet-dev.colorado.edu/html/concentration/1.6.0-dev.1

**Comment by pixelzoom** [comment]
Further work on this is on hold until ComboBox layout https://github.com/phetsims/sun/issues/788 is fixed.

**Comment by pixelzoom** [comment]
ComboBox was fixed in https://github.com/phetsims/sun/issues/788 and is behaving correctly.

@arouinfar ready for review.

**Comment by arouinfar** [comment]
@pixelzoom the dynamic layout all behaves as expected. The only hiccup was in finding the strings for the labels that are followed by colons -- Solute, Evaporation, Wavelength, and Solution. I see `labelText` in the tree, but no linked property below. For example:
<img width="241" alt="image" src="https://user-images.githubusercontent.com/8419308/206096681-2f8a5c0e-83d5-427c-93dc-da513abbe8e7.png">

I think the issue is that it's a combined string. The colon isn't in the `StringProperty` so things don't link up easily.

**Comment by pixelzoom** [comment]
@arouinfar please review these elements in Studio, and let me know if this is what you had in mind.

"Evaporation:" -> evaporationPanel.labelText.stringProperty
"Solute:" -> soluteComboBox.labelText.stringProperty
"Solutions:" -> solutionPanel.solutionComboBox.labelText.stringProperty"
"Wavelength:" -> wavelengthPanel.labelText.stringProperty

I can't address "Concentration:" yet due to https://github.com/phetsims/beers-law-lab/issues/270.

Please assign back to me after review.

**Comment by arouinfar** [comment]
Thanks @pixelzoom. I now see the dependencies linked up, which is what I expected. For example:

<img width="299" alt="image" src="https://user-images.githubusercontent.com/8419308/206265069-988e1338-6e14-4111-aed7-6c49d2fc9ba1.png">

> I can't address "Concentration:" yet due to https://github.com/phetsims/beers-law-lab/issues/270.
>
> Please assign back to me after review.

Good to know, back to you.

**Comment by pixelzoom** [comment]
Note to self:  
- [ ] Address "Concentration:" after resolving #270.

**Comment by pixelzoom** [comment]
I moved instrumentation of the "Concentration:" label to https://github.com/phetsims/beers-law-lab/issues/297.  So closing this issue.
